### PR TITLE
Do not use snippets after import statements

### PR DESCRIPTION
**Pull request by sadovnychyi**
#83

### CODE REVIEW

**Comment by patrys** [comment]
Why not limit the snippet to completion.name instead? Then we can always return a snippet and won't have to support multiple code paths.

**Comment by sadovnychyi** [comment]
As I remember you have done it in #85.

**Comment by sadovnychyi** [comment]
I just did it on top of master, so this two PRs are independent.

File: lib/completion.py [hunk 1]
```diff
@@ -174,12 +176,18 @@ def _set_request_config(self, config):
             if path and path not in sys.path:
                 sys.path.insert(0, path)
 
+    def _get_completion_context(self, request):
+      """Returns string with everything before cursor on current line.
+      """
+      return request['source'].splitlines()[request['line']][:request['column']]
```
Comment (patrys) [inline lib/completion.py]: Might use some safeguarding with `request.get(…)`?

Comment (sadovnychyi) [inline lib/completion.py]: Probably better just catch all IndexError and KeyError.

**Pull request merged**